django_admin_reset/admin.py

```diff
@@ -58,14 +59,35 @@ def get_context_data(self, **kwargs):
         return context
 
 
+if UserModel.USERNAME_FIELD == 'email':
+    PERSONAL_INFO_FIELDS = ('first_name', 'last_name')
+    LIST_DISPLAY = (UserModel.USERNAME_FIELD, 'first_name', 'last_name', 'is_staff')
+else:
+    PERSONAL_INFO_FIELDS = ('first_name', 'last_name', 'email')
+    LIST_DISPLAY = (UserModel.USERNAME_FIELD, 'email', 'first_name', 'last_name', 'is_staff')
+
+
 class PasswordResetUserAdmin(UserAdmin):
     add_form = UserCreationForm
     add_form_template = 'admin/add_user_form.html'
+    fieldsets = (
+        (None, {'fields': (UserModel.USERNAME_FIELD, 'password')}),
+        (_('Personal info'), {'fields': PERSONAL_INFO_FIELDS}),
+        (_('Permissions'), {
+            'fields': ('is_active', 'is_staff', 'is_superuser', 'groups', 'user_permissions'),
+        }),
+        (_('Important dates'), {'fields': ('last_login', 'date_joined')}),
+    )
     add_fieldsets = (
-        (None, {'classes': ('wide',), 'fields': ('username',)}),
+        (None, {
+            'classes': ('wide', ),
+            'fields': (UserModel.USERNAME_FIELD,) + UserModel.REQUIRED_FIELDS
```

Comment:
The tests are failing on this line, because `UserModel.REQUIRED_FIELDS` is a list by default: https://travis-ci.org/tiborhari/django-admin-reset/builds/653628076